addon/templates/components/bootstrap-inputs/-email.hbs

```diff
@@ -0,0 +1,54 @@
+{{#bootstrap/control-wrapper
+  classNames=classNames
+  errors=errors
+  srOnly=srOnly
+  label=label
+  as |control|
+}}
+  {{#if hasBlock}}
+    {{yield
+      (hash
+        label=(component control.label for=control.inputId)
+        control=control
+        input=(component 'bootstrap/-input'
+          value=value
+          elementId=control.inputId
+          type='email'
+          disabled=disabled
+          autocomplete=autocomplete
+          maxlength=maxlength
+          minlength=minlength
+          multiple=multiple
+          pattern=pattern
+          placeholder=placeholder
+          readonly=readonly
+          required=required
```

Comment:
This might be a good opportunity to add a visual indicator of requirement to our inputs (red star next to the label for example). I know we haven't used them too often, but there have been instances where they were beneficial. Just a though

Reply:
There is no opinions from  bootstrap 4 regarding this, but I think we could add our own if we want.